androidlibrary_lib/src/androidTest/java/org/opendatakit/database/data/KeyValueStoreEntryTest.java

```diff
@@ -0,0 +1,217 @@
+package org.opendatakit.database.data;
+
+import static org.junit.Assert.*;
+
+import android.os.Parcel;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@SuppressWarnings("ALL")
+@RunWith(AndroidJUnit4.class)
+public class KeyValueStoreEntryTest {
+
+    private KeyValueStoreEntry ENTRY_1;
```

Comment:
The general convention is that only constants are capitalized (things you expect to not need to change).

Please fix everywhere